js/components/scrubberBar.js

```diff
@@ -67,7 +68,7 @@ var ScrubberBar = React.createClass({
   handlePlayheadMouseDown: function(evt) {
     if (this.props.controller.state.screenToShow == CONSTANTS.SCREEN.AD_SCREEN) return;
     this.props.controller.startHideControlBarTimer();
-    if (evt.target.className.match("playhead") && evt.type !== "mousedown") {
+    if (evt.target.className.match("oo-playhead") && evt.type !== "mousedown") {
```

Comment:
FYI @chibzu this could have been a possible bug or 'fixing' it could introduce a bug.

After update, I did not notice any bugs while testing.